frontend/pages/hosts/details/cards/Software/HostSoftwareTable/HostSoftwareTable.tsx

```diff
@@ -102,6 +102,13 @@ const HostSoftwareTable = ({
         routeTemplate,
         queryParams: newParams,
       });
+      const prevYScroll = window.scrollY;
+      setTimeout(() => {
+        window.scroll({
+          top: prevYScroll,
+          behavior: "smooth",
```

Comment:
Any reason why we wouldn't do `"instant"`?

Reply:
I played around a bit, the immediate snap options like that end up looking like a UI flash on page load, smooth seems better to me

Reply:
sgtm cc: @rachaelshaw